py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -338,6 +338,12 @@ def format_data(self,
                     "z": row['Z'],
                 }
                 args["log_lambda"] = np.log10(spec['WAVELENGTH'])
+                if "TILEID" in row:
+                    args["tile"] = row['TILE']
```

Comment:
Isn't it called TILEID?

Reply:
Yes, I have fixed the bug (fixed it in a commit). Should we rename the `tile` member in `DesiForest` to `tileid` as well?

Reply:
maybe that would be good for consistency... Esp given that e.g. eBOSS did have a PLATE instead of TILEID anyway, so it's a DESI specific thing.